src/forwarder/index.js

```diff
@@ -54,8 +55,8 @@ function log(message, severity = 'DEBUG', payload) {
  *
  * For testing purposes - uses a fixed message.
  *
- * @param {Request} req
- * @param {Response} res
+ * @param {express.Request} req
+ * @param {express.Response} res
  */
 exports.forwardFromHTTP = async (req, res) => {
```

Comment:
Do we want to use `module.exports` here, to match `scaler-core`?

Reply:
Done